From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@jboeshart__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (jaboes). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

Ja-Dunn commented:
@jboeshart - Thanks for your contribution. We are in the process of refactoring the Azure technical articles, so we can’t merge this PR. When the refactoring is complete, we can give you a link to the new location of the article you updated so you can resubmit your update, if you want to.

jboeshart commented:
@Ja-Dunn that works for me. Let me know the new URL when you have it, I'll also keep an eye out post-migration as well for the new doc and will resubmit.

gbowerman commented:
#sign-off 
thanks @jboeshart

ktoliver commented:
@jboeshart - Thanks for your contribution to the Azure docs. On November 16th, we announced the availability of our Azure technical documentation in a new location on [docs.microsoft.com](https://docs.microsoft.com). You can [read about it on our team blog](https://docs.microsoft.com/teamblog/docs-november-update). This change means the article source lives in a new GitHub repository. Because there are some changes in the source markdown as a result of the move, we can’t pull your contribution over to the new repo without creating a ton of merge conflicts. We invite you to resubmit the change (if you are willing) to the new version of the article at [https://docs.microsoft.com/en-us/azure/virtual-machine-scale-sets/virtual-machine-scale-sets-deploy-app](https://docs.microsoft.com/en-us/azure/virtual-machine-scale-sets/virtual-machine-scale-sets-deploy-app). Thanks for your understanding.

jboeshart commented:
PR created in new repo. 

https://github.com/Microsoft/azure-docs/pull/69

mrbullwinkle committed:
hockeyapp deprecation

Commit:
First draft of permissions content.

Commit:
Merge remote-tracking branch 'MicrosoftDocs/master' into permissions

Commit:
Updated TOC to reference permissions doc.

v-viinde committed:
USER STORY 1241036 SaaS App Tutorial:OrgChart Now Update

TsuyoshiUshio committed:
Adding Event Grid Publishing first idea

CelesteDG committed:
Broken link fix and some copyedit changes

Fixed broken link to a topic and made some copyedit/formatting edits

Commit:
Addressing acrolinx feedback.

MicrosoftGuyJFlo committed:
[Azure AD] SSPR - Add GTP deployment project plan to docs

CelesteDG committed:
Merge branch 'permissions' into patch-3

TsuyoshiUshio committed:
fix the misspelled and grammar correction

TsuyoshiUshio committed:
fix: Event Grid topic

TsuyoshiUshio committed:
fix: instance Id

TsuyoshiUshio committed:
fix: Grammar section

TsuyoshiUshio committed:
fix: style section

TsuyoshiUshio committed:
fix: Converstational Tone 1 of 2

TsuyoshiUshio committed:
fix: Press Create

TsuyoshiUshio committed:
fix: Style

TsuyoshiUshio committed:
fix: This json style

TsuyoshiUshio committed:
fix: eventType of the Event Schema was wrong

TsuyoshiUshio committed:
add some fix and Local debugging instrcution, and the next steps

TsuyoshiUshio committed:
Remove unnesessary pngs

TsuyoshiUshio committed:
fix: Grammar expression

tdykstra committed:
title and metadata

tdykstra committed:
copy edit

MicrosoftGuyJFlo committed:
Updates from PG

TsuyoshiUshio committed:
Merge pull request #2 from tdykstra/patch-38

copy edit

TsuyoshiUshio committed:
Help customer understand which log is that?

TsuyoshiUshio committed:
Changing the sentence to fit

TsuyoshiUshio committed:
fix: style

TsuyoshiUshio committed:
fix: changing the url of Ngrok

KumudD committed:
mvc update

KumudD committed:
fixed file

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into trafficmanagersamples0425

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into trafficmanagersamples0425

KumudD committed:
refactored toc and added samples

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into trafficmanagersamples0425

KumudD committed:
added yaml files

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into trafficmanagersamples0425

KumudD committed:
added yaml toc and index files

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into trafficmanagersamples0425

KumudD committed:
fixed links

KumudD committed:
fixed foramtting

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into trafficmanagersamples0425

DCtheGeek committed:
Language change for GH Issue #7536

jackrichins committed:
Update Storage Files, Tables, and Queues

Tables and Queues do support Service Managed Keys and Files now support Customer Managed in Key Vault keys.

Commit:
Connection monitor

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into connection-monitor

KumudD committed:
addressed PR review feedback

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into trafficmanagersamples0425

Commit:
Merge choice help content into services and technologies

tfitzmac committed:
added new built in policies

Commit:
Created new index.yml

Commit:
save

mattbriggs committed:
Python SDK

mattbriggs committed:
adding changed files

KumudD committed:
updated TOC and Landing pages as per feedback

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into trafficmanagersamples0425

KumudD committed:
updated tutorial list to map to TOC list

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into trafficmanagersamples0425

KumudD committed:
updated tutorial list

mmacy committed:
[ACR] geo-rep refresh

TsuyoshiUshio committed:
Add gray line for which has white edge image

TsuyoshiUshio committed:
add preview as the title

Commit:
Updates.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into connection-monitor

mayanknayar committed:
Using ExpressRoute with Site Recovery

Adding guidance on using ExpressRoute with different Site Recovery
scenarios

mayanknayar committed:
Edits for ER with ASR

Edits for clarity

mayanknayar committed:
Link corrections for ER with ASR

Link corrections for ER with ASR

Commit:
Fixes.

mayanknayar committed:
Link and clarity improvements for ER with ASR

Link and clarity improvements for ER with ASR

Commit:
Update.

mmacy committed:
[ACR] geo-rep tut 3 refresh

Commit:
Update data-lake-analytics-quota-limits.md

Commit:
Fixes.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into connection-monitor

TsuyoshiUshio committed:
adding md at the end of the url

mmacy committed:
[ACR] typo fix

cherylmc committed:
Update vpn-gateway-sample-site-to-site-powershell.md

jackrichins committed:
Addressed Style Issues

Addressed style comments from Scorecard for my changes.

mrbullwinkle committed:
workaround to public prs not merging

KumudD committed:
toc restructure

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into CDSindex

dlepow committed:
Fixes MicrosoftDocs/azure-docs#7736

v-stadam committed:
overview fixes

jackrichins committed:
Improved wording

Further improved clarity.

Commit:
Update iot-services-and-technologies.md

Merged the compare section into the  solutions section.

PRMerger12 committed:
Merge pull request #39528 from saveenr/patch-33

Update data-lake-analytics-quota-limits.md

mmacy committed:
[ACR] tut image 1 update

PRMerger13 committed:
Merge pull request #39389 from jackrichins/patch-3

Update Storage Files, Tables, and Queues

v-shils committed:
Merge pull request #39072 from KumudD/trafficmanagersamples0425

Traffic Manager TOC refactor

sethmanheim committed:
Fixing metadata values per cleanup project

rmca14 committed:
Merge pull request #39544 from v-stadam/Overview_fix

overview fixes

mrbullwinkle committed:
Merge branch 'master' into mrb_04_16_2018_hockeyapp

PRMerger20 committed:
Merge pull request #39543 from dlepow/batchcontainer

Batch: containers article update

mrbullwinkle committed:
modify wording.

mrbullwinkle committed:
mod

mrbullwinkle committed:
additional modifications

Commit:
Merge pull request #39549 from sethmanheim/meta-fix

Fixing metadata values per cleanup project

douglaslMS committed:
Made PM changes for custom setup.

mmacy committed:
[ACR] update output blocks

daveirwin1 committed:
Linux agent 9.5.1

PRMerger10 committed:
Merge pull request #39535 from cherylmc/patch-2

Update vpn-gateway-sample-site-to-site-powershell.md

jhutchings1 committed:
Merge pull request #1 from CelesteDG/patch-3

Broken link fix and some copyedit changes

Commit:
Fixing author metadata

PRMerger13 committed:
Merge pull request #37944 from mrbullwinkle/mrb_04_16_2018_hockeyapp

hockeyapp changes

itechedit committed:
Merge pull request #39407 from ClareMSYanGit/cyan-master-working-iot

Merge choice help content into services and technologies

PRMerger14 committed:
Merge pull request #39567 from daveirwin1/master

Linux agent 9.5.1

tfitzmac committed:
changed resource type

PRMerger16 committed:
Merge pull request #39545 from mrbullwinkle/mrb_04_30_2018_public_pull_request_fix

workaround to public prs not merging

rboucher committed:
Change metric retention from 30 to 90 days

per the pricing page and and issue https://github.com/MicrosoftDocs/azure-docs/issues/7880

Commit:
Merge pull request #39568 from rboucher/patch-9

Change metric retention from 30 to 90 days

DCtheGeek committed:
Updating mode defaults for GH Issue #7871

mmacy committed:
[ACR] match digests

cherylmc committed:
update

Commit:
misc. updates for freshness

mimckitt committed:
Fixes MicrosoftDocs/azure-docs#7885

The image offer SQL2016-WS2016 doesn´t work. Removing from list

tamram committed:
removing warning per PM request

rwike77 committed:
adding info on reverse proxy port

Commit:
Fixes.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into connection-monitor

v-shils committed:
Merge pull request #39416 from tfitzmac/0427policy

added new built in policies

v-shils committed:
Merge pull request #39488 from mmacy/acr-georep-refresh

[ACR] geo-rep refresh

itechedit committed:
Merge pull request #39424 from wiazur/CDSindex

Created new index.yml

PRMerger15 committed:
Merge pull request #39576 from tamram/tamram-0430

removing page blob warning per PM request

PRMerger15 committed:
Merge pull request #39566 from mmacy/acr-build-tutorial-update

[ACR] update output blocks

Commit:
slight rewording.

TsuyoshiUshio committed:
Adding durable-functions-event-publishing.md to TOC file

mattbriggs committed:
update python and go

PRMerger20 committed:
Merge pull request #39577 from rwike77/dotnettutorial2

adding info on reverse proxy port

PRMerger20 committed:
Merge pull request #38621 from MicrosoftGuyJFlo/PGUserStory1235202BacklogItem405582

[Azure AD] SSPR - Add GTP deployment project plan to docs

tfosmark committed:
Merge pull request #39563 from douglaslMS/adf-custom-setup

Made PM changes for custom setup.

tfosmark committed:
Merge pull request #39541 from KumudD/vpngatewaytocupdate

toc restructure: vpn gateway

Commit:
Clarifying SQL DB and SQL DW are supported in resource moves

TrishaMc committed:
Merge pull request #39517 from mayanknayar/master

Using ExpressRoute with Site Recovery

v-shils committed:
Merge pull request #38541 from TsuyoshiUshio/doc/eventgridpublishing

Durable Functions Automatic Event Grid publishing documentation

PRMerger13 committed:
Merge pull request #39578 from Blackmist/freshness

misc. updates for freshness

twooley committed:
Updated next steps

PRMerger16 committed:
Merge pull request #39597 from TwoUnder/master

Clarifying SQL DB and SQL DW are supported in resource moves

itechedit committed:
Merge pull request #39397 from jimdial/connection-monitor

Connection monitor

PRMerger17 committed:
Merge pull request #39570 from DCtheGeek/dmc-policy-patch3

Updating mode defaults for GH Issue #7871

douglaslMS committed:
Added link to Channel 9 video demo.

PRMerger19 committed:
Merge pull request #39575 from MicahMcKittrick-MSFT/patch-25

Fixes MicrosoftDocs/azure-docs#7885

PRMerger19 committed:
Merge pull request #39601 from twooley/master

Updated next steps

TrishaMc committed:
Merge pull request #39449 from mattbriggs/mdb_20180428_apiprofile-python

Python SDK

Commit:
Merge pull request #39604 from douglaslMS/adf-databricks-video

Added link to Channel 9 video demo.

GitHubber17 committed:
Merge pull request #38375 from jhutchings1/permissions

Added overview of permissions in Azure AD

GitHubber17 committed:
Merge pull request #38397 from v-viinde/orgchartnowupdate

USER STORY 1241036 SaaS App Tutorial:OrgChart Now Update

tamram committed:
fix for issue 7881 - blobContinuationToken set twice

douglaslMS committed:
Added PM video link to Channel 9 video.

PRMerger12 committed:
Merge pull request #39370 from DCtheGeek/dmc-policy-patch1

Language change for GH Issue #7536

iainfoulds committed:
[VM] Fixes MicrosoftDocs/azure-docs#7775

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

PRMerger15 committed:
Merge pull request #39609 from tamram/tamram-0430a

fix for issue 7881 - blobContinuationToken set twice

iainfoulds committed:
[VM] Update metadata

PRMerger17 committed:
Merge pull request #39610 from douglaslMS/adf-contint-video

Added PM video link to Channel 9 video.

v-alje committed:
Merge pull request #39612 from MicrosoftDocs/FromPublicRepo

From public repo

PRMerger18 committed:
Merge pull request #39579 from cherylmc/samples

update

PRMerger19 committed:
Merge pull request #39611 from iainfoulds/patch-4

[VM] Linux SSL web server update
